.eslintrc.json

```diff
@@ -1,37 +1,48 @@
 {
-    "extends": "google",
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-      "ecmaVersion": "2017",
-      "sourceType": "module"
-    },
-    "plugins": [
-      "@typescript-eslint"
-    ],
-    "rules": {
-      "max-len": ["error", {
+  "extends": ["plugin:prettier/recommended", "google"],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "2017",
+    "sourceType": "module"
+  },
+  "plugins": ["@typescript-eslint"],
+  "rules": {
+    "indent": "off",
+    "quote-props": "off",
+    "max-len": [
+      "error",
+      {
         "ignoreStrings": true,
```

Comment:
i suspect because we have `ignoreUrls` now, this one can be removed and any outliers fixed. its a _very_ greedy "ignore" in that it ignores *any* line containing a string (including ifs, etc) so i prefer to not have it

Reply:
I can look at removing this.